src/manifolds/VectorBundle.jl

```diff
@@ -617,23 +529,37 @@ Notation:
     canonical projection of that vector bundle $B$.
     Similarly, $q = (x_q, V_q)$.
 
-The logarithmic map is calculated as
+The inverse retraction is calculated as
 
 $\log_p q = (\log_{x_p}(x_q), V_{\log} - V_p)$
```

Comment:
Maybe also adapt the first part here, this make it clearer that the retraction works as element wise things.

```suggestion
$\operatorname{retr}_p q = (\log_{x_p}(x_q), V_{\log} - V_p)$
```

should the retraction maybe have a parameter, which vector transport to use (and the default is `default_vector_transport(M)` when constructing it)?

Reply:
Currently it uses the vector transport stored in the `VectorBundle` object. What it lacks currently is the ability to define the inverse retraction on the manifold that should be used (currently it just does `log`). I will check documentation a bit later.

Reply:
Ah, that sounds also fine, I was not aware, the bundle would store its transport.